src/jobs/microk8s_install.yml

```diff
@@ -31,6 +31,14 @@ parameters:
       global:
         imagePullSecrets:
           - name: docker-credentials
+        play:
```

Comment:
these are only for codacy website and specific components.
Should not pollute orbs

Reply:
You're right. If/When these change we would have to bump the orb or just write the parameter in the workflows them anyway. It makes no sense to have this here. I'll just remove  the default.